lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/device/connection/DeviceConnectionManager.java

```diff
@@ -125,6 +130,25 @@ Establish connection with device (future will be set with GnmiDeviceManager for
             executorService);
     }
 
+    private Gnmi.CapabilityResponse getExtensionYangModelsList(final DeviceConnection deviceConnection) {
```

Comment:
rename `getExtensionYangModelsList` ... -> `getExtensionCapabilitiesList`
you are working with capabilities in method, and return type of method is also capability, mixing of names may confuse reader